packages/functional-tests/tests/settings/changeEmail.spec.ts

```diff
@@ -140,7 +140,9 @@ test.describe('severity-1 #smoke', () => {
       await signin.fillOutEmailFirstForm(initialEmail);
       await signin.fillOutPasswordForm(newPassword);
 
-      await expect(settings.primaryEmail.status).toHaveText(credentials.email);
+      await expect(settings.primaryEmail.status).toHaveText(credentials.email, {
+        timeout: 1000,
+      });
```

Comment:
This will override the default timeout of [10 seconds](https://github.com/mozilla/fxa/blob/main/packages/functional-tests/playwright.config.ts#L43) to 1 second, making the test less resilient.

<img width="1619" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/9f159206-54af-4453-adc5-1607773fae51">